src/main/java/com/redhat/devtools/intellij/common/actions/TknAction.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
```

Comment:
The common package is supposed to be extracted from this plugin and should contain only generic stuff. So every piece of code that is Tekton specific must be moved under the tektoncd sub packages